build.gradle.kts

```diff
@@ -37,14 +37,14 @@ java {
 }
 
 dependencies {
-    implementation(group = "net.msrandom", name = "minecraft-codev-forge", version = "0.5.29")
-    implementation(group = "net.msrandom", name = "minecraft-codev-fabric", version = "0.5.29")
-    implementation(group = "net.msrandom", name = "minecraft-codev-mixins", version = "0.5.29")
-    implementation(group = "net.msrandom", name = "minecraft-codev-runs", version = "0.5.29")
-    implementation(group = "net.msrandom", name = "minecraft-codev-access-widener", version = "0.5.29")
-    implementation(group = "net.msrandom", name = "minecraft-codev-remapper", version = "0.5.29")
-    implementation(group = "net.msrandom", name = "minecraft-codev-decompiler", version = "0.5.29")
-    implementation(group = "net.msrandom", name = "minecraft-codev-includes", version = "0.5.29")
+    implementation(group = "net.msrandom", name = "minecraft-codev-forge", version = "0.5.30")
```

Comment:
This is requiring a version that should be released after this(not before)

Reply:
Resolving; 0.5.30 was published now but with different changes than this PR expects.